uavcan/time/TAIInfo.0.1.uavcan

```diff
@@ -0,0 +1,63 @@
+#
+# This data types defines constants and runtime values pertaining to the International Atomic Time, also known as TAI.
+# See https://en.wikipedia.org/wiki/International_Atomic_Time.
+#
+# The relationship between the three major time systems -- TAI, GPS, and UTC -- is as follows:
+#
+#   TAI = GPS + 19 seconds
+#   TAI = UTC + LS + 10 seconds
```

Comment:
Probably. It doesn't seem to be going away soon. I don't know much about that system though.

Do we want it added now, or can it be postponed until another review?

Reply:
My understanding is that there is new investment in Loran in the face of certain regional conflicts where GPS jamming would otherwise cripple a defense system (see https://arstechnica.com/gadgets/2017/08/radio-navigation-set-to-make-global-return-as-gps-backup-because-cyber/)

Ah, minutiae! How I love thee.

This can wait. Just a note for future consideration.